crates/fluvio-hub-util/src/hubaccess.rs

```diff
@@ -307,10 +312,9 @@ fn get_hubref() -> Option<String> {
         return None; // use default
     }
     let hubref_url = format!("{fcremote}/api/v1/hubref");
-    let reply: std::result::Result<String, surf::Error> = run_block_on(async {
-        let req = surf::get(hubref_url);
-        let mut res = req.await?;
-        let reply: ReplyHubref = res.body_json().await?;
+    let reply: std::result::Result<String, reqwest::Error> = run_block_on(async {
+        let res = reqwest::get(hubref_url).await?;
+        let reply: ReplyHubref = res.json().await?;
```

Comment:
to use `.json()` here you need to enable the feature flag in the `Cargo.toml`

`reqwest = { ..., features = [..., "json"] }`

Reply:
Oddly this wasn't erroring locally, but will do.